job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/triggers/GitHubPullRequestBuilderCommitStatusContext.groovy

```diff
@@ -31,6 +32,13 @@ class GitHubPullRequestBuilderCommitStatusContext implements Context {
     void startedStatus(String startedStatus) {
         this.startedStatus = startedStatus
     }
+    
+    /**
+     * Use a custom url instead of the job default.  Use --none-- to specify an empty URL.
```

Comment:
add a `@since 1.40` annotation. Do not mention --none--, it's really an issue of the ghprb plugin and should be fixed there, but should not be mentioned here.